packages/api-client/src/components/ApiClient/Request/RequestBody.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
 import { CodeMirror } from '@scalar/use-codemirror'
+import { computed } from 'vue'
```

Comment:
Is this being used in here?

Reply:
no it's not, I can remove it good catch!